config/genesis.go

```diff
@@ -0,0 +1,82 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+
+	apiConfig "github.com/spacemeshos/go-spacemesh/api/config"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/hash"
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+const (
+	DefaultGenesisConfigFileDir  = "data"
```

Comment:
so given that spacemesh is configured with datadir - /state/ . genesis config will saved into /state/data/genesis.conf. does it make sense? why is data needed?

Reply:
Not needed really. Removed.